meson.build

```diff
@@ -293,7 +312,21 @@ if get_option('enable-blas')
     blas_root = meson.build_root() / 'openblas'
     blas_dep = declare_dependency(include_directories: [ 'openblas/include' ])
   else
-    blas_dep = dependency('openblas')
+    blas_dep = dependency('openblas', required : false)
+  
+    if not blas_dep.found()
+      blas_options = cmake.subproject_options()
+      if host_machine.system() == 'windows'
+        blas_options.append_compile_args('c', windows_compile_args)
+        blas_options.append_compile_args('cpp', windows_compile_args)
+      endif
+      blas_options.add_cmake_defines({'BUILD_TESTING': false})
+      blas_options.add_cmake_defines({'BUILD_BENCHMARKS': false})
+      blas_options.add_cmake_defines({'BUILD_SHARED_LIBS': false})
+      blas_options.add_cmake_defines({'BUILD_WITHOUT_LAPACK': true})
+      blas_subproject = cmake.subproject('openblas', options: blas_options, required: true)
```

Comment:
Didn't ever test it but worth checking if defining those in native [cmake] section works